open_instruct/grpo_vllm_thread_ray_gtrl.py

```diff
@@ -634,8 +659,10 @@ def from_pretrained(
             attn_implementation="flash_attention_2",
             use_cache=False,
         )
+        self.policy_vocab_size = self.policy.config.vocab_size
         disable_dropout_in_model(self.policy)
         self.policy.gradient_checkpointing_enable()
+        # from deepspeed.ops.adam import DeepSpeedCPUAdam, FusedAdam
```

Comment:
prolly remove?

Reply:
I am inclined to keep it in case we need to comment it out and test it out something (we wouldn't know where FusedAdam comes from)